de.metas.endcustomer.mf15/de.metas.endcustomer.mf15.base/src/main/jasperreports/de/metas/docs/purchase/inout/report.properties

```diff
@@ -8,6 +8,7 @@ vonuns = von uns
 showfooter=Y
 showdetailsfooter=Y
 details_product_overflow=N
+showdescription=Y
```

Comment:
hi @metas-lc, 
for such not-selfexplanatory properties, please generally add comments to the properties files, similar to the ones we would add to java files (just starting with `#` ;-)). 

I mean, please write the issue# or -URL if there is any, and a short description about what the parameter does. E.g. where it's not clear which description is shown where.